<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addRroDialog</class>
 <widget class="QDialog" name="addRroDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/Icon/ekkr.png</normaloff>:/resource/Icon/ekkr.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>ЗН:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>POS ID:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Экваер:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditIdDev"/>
   </item>
   <item row="5" column="1">
    <widget class="QRadioButton" name="radioButtonUkr">
     <property name="text">
      <string>УкрКард</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditFn"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>ID DEV:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="comboBoxTerminal">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string>Выберите терминал</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditZn"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="comboBoxModel">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>ФН:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QDateTimeEdit" name="dateTimeEditPers">
     <property name="date">
      <date>
       <year>2010</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Модель РРО:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QSpinBox" name="spinBoxPos">
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Дата
персонализации:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Терминал:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QRadioButton" name="radioButtonIks">
     <property name="text">
      <string>ИКС</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBoxRro">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QRadioButton" name="radioButtonNoServise">
     <property name="text">
      <string>Не обслуживаем</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxTerminal</tabstop>
  <tabstop>spinBoxPos</tabstop>
  <tabstop>comboBoxModel</tabstop>
  <tabstop>lineEditZn</tabstop>
  <tabstop>lineEditFn</tabstop>
  <tabstop>radioButtonUkr</tabstop>
  <tabstop>radioButtonIks</tabstop>
  <tabstop>lineEditIdDev</tabstop>
  <tabstop>dateTimeEditPers</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBoxRro</sender>
   <signal>accepted()</signal>
   <receiver>addRroDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBoxRro</sender>
   <signal>rejected()</signal>
   <receiver>addRroDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
